packages/react-components/src/theme-provider/src/ThemeContext.ts

```diff
@@ -1,13 +1,13 @@
 import { createContext, useContext } from "react";
-import type { ColorScheme, Theme } from "./ThemeProvider";
+import type { ColorScheme, ColorSchemeOrSystem, Theme } from "./ThemeProvider";
 
 export interface ThemeContextType {
-    theme?: Theme;
-    colorScheme?: ColorScheme;
-    setColorScheme?: (newColorScheme: ColorScheme) => void;
+    theme: Theme;
```

Comment:
As mentionned in the issue, I think `theme` and `colorScheme` should remain optional

Reply:
Sorry, I thought it was the component interface, closing.